web/src/features/moreCast2/pages/MoreCast2Page.tsx

```diff
@@ -74,7 +80,11 @@ const MoreCast2Page = () => {
 
   const fetchStationPredictions = () => {
     const stationCodes = fireCenter?.stations.map(station => station.code) || []
-    dispatch(getModelStationPredictions(stationCodes, modelType, fromDate.toISODate(), toDate.toISODate()))
+    if (modelType == 'YESTERDAY') {
```

Comment:
Could we make this
`if (modelType === ModelChoice.Yesterday)`?

Reply:
Sure, it's still type safe this way since the `modelType` is type aliased by string unions but I think I can do that as well.

Reply:
Addressed in: https://github.com/bcgov/wps/pull/2684/commits/9fce764a98d9223514e3efd53105b9abab5feba9